cmd/buildah/prune.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	internalParse "github.com/containers/buildah/internal/parse"
+	buildahcli "github.com/containers/buildah/pkg/cli"
+	"github.com/containers/buildah/pkg/parse"
+	"github.com/containers/common/libimage"
+	"github.com/hashicorp/go-multierror"
+	"github.com/spf13/cobra"
+)
+
+type pruneOptions struct {
+	force bool
+}
+
+func init() {
+	var (
+		pruneDescription = "\n  Clean up build cache, intermediate images and mount cache."
```

Comment:
Renive tge \n and just put this on next line, like it is in other commands.

Reply:
I would expect this to concentrate on images rather then cache

Perhaps
```
Cleanup intermediate images as well as build and mount cache
```

Reply:
Done.